internal/pkg/deploy/cloudformation/stack/transformers.go

```diff
@@ -688,9 +688,18 @@ func (s *LoadBalancedWebService) convertNetworkLoadBalancer() (networkLoadBalanc
 		return networkLoadBalancerConfig{}, fmt.Errorf(`convert "nlb.alias" to string slice: %w`, err)
 	}
 
+	// TODO(Aiden): remove when NetworkLoadBalancer is forcibly updated
+	var udpListenerExists bool
+	for _, listener := range listeners {
+		if listener.Protocol == "UDP" {
```

Comment:
A more robust comparison would be [`strings.EqualFold`](are we sure that `listener.Protocol` wouldn't be `"udp"` in lowercase? ) that do not assume the listener's protocols will be uppercase by this point! This way, when the other packages change in the future such that this assumption no longer holds, we won't need to touch the code here as a result. 

But I think this is fine - as long as we are sure that they are always uppercase - because these codes are going to be removed 2 release later, so maintainability is generally not a concern.